### PR TITLE
Migrated getChannel to Push SDK

### PR DESCRIPTION
**Pull request by hrithik73**
Changed getChannel method to get the channels list from push sdk using userPushSDKInstance.channel.list() method

### CODE REVIEW

**Comment by hrithik73** [comment]
#1434

**Review by HarshRajat — APPROVED** [review]

File: src/segments/ViewChannels.tsx [hunk 1]
```diff
@@ -88,14 +79,15 @@ function ViewChannels({ loadTeaser, playTeaser, minimal }) {
 
   // to fetch initial channels and logged in user data
   const fetchInitialsChannelMeta = async () => {
-    // fetch the meta of the first `CHANNELS_PER_PAGE` channels
-    const channelsMeta = await getChannels({
+
```
Comment (0xNilesh) [inline src/segments/ViewChannels.tsx:82]: Add try-catch here

File: src/segments/ViewChannels.tsx [hunk 2]
```diff
@@ -110,11 +102,11 @@ function ViewChannels({ loadTeaser, playTeaser, minimal }) {
   // load more channels when we get to the bottom of the page
   const loadMoreChannelMeta = async (newPageNumber: any) => {
     const startingPoint = newPageNumber * CHANNELS_PER_PAGE;
```
Comment (0xNilesh) [inline src/segments/ViewChannels.tsx:104]: Add try-catch in this function as well

File: src/segments/ViewChannels.tsx (hunk 1 above)
Comment (hrithik73) [reply src/segments/ViewChannels.tsx]: added !

File: src/segments/ViewChannels.tsx (hunk 2 above)
Comment (hrithik73) [reply src/segments/ViewChannels.tsx:110]: added !

**Pull request merged**